Discussion/DiscussionTests/Presentation/Posts/PostViewModelTests.swift

```diff
@@ -174,7 +185,13 @@ final class PostViewModelTests: XCTestCase {
         let router = DiscussionRouterMock()
         let config = ConfigMock()
         var result = false
-        let viewModel = PostsViewModel(interactor: interactor, router: router, config: config)
+        let viewModel = PostsViewModel(
```

Comment:
As per my previous comment, the object is used in multiple methods so we can initialize it in the `setup` method and use it in the whole class in numerous methods.